jenkins_job_wrecker/modules/handlers.py

```diff
@@ -16,6 +16,8 @@ def gen_yml(self, yml_parent, data):
                 for setting in settings:
                     key, value = setting
                     if key in yml_parent:
+                        if len(value) == 0:
```

Comment:
I prefer `if not value:` instead of `if len(foo) == 0`

Reply:
Cool, I didn't realize that an empty list returned false in conditionals. Learn something new everyday 👍